changelog/Microsoft.IdentityProtectionServices.json

```
@@ -52,6 +52,80 @@
 			"WorkloadArea": "Identity and access",
 			"SubArea": "Identity and sign-in"
 		},
+		{
+		      "ChangeList": [
+			{
+			  "Id": "68fb7e36-efee-4e07-a53c-06325d9e1e91",
+			  "ApiChange": "Enumeration",
+			  "ChangedApiName": "conditionalAccessExternalTenantsMembershipKind",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **conditionalAccessExternalTenantsMembershipKind** enumeration type.",
+			  "Target": "conditionalAccessExternalTenantsMembershipKind"
+			},
+			{
+			  "Id": "68fb7e36-efee-4e07-a53c-06325d9e1e91",
+			  "ApiChange": "Enumeration",
+			  "ChangedApiName": "conditionalAccessGuestOrExternalUserTypes",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **conditionalAccessGuestOrExternalUserTypes** enumeration type.",
+			  "Target": "conditionalAccessGuestOrExternalUserTypes"
+			},
+			{
+			  "Id": "68fb7e36-efee-4e07-a53c-06325d9e1e91",
+			  "ApiChange": "Resource",
+			  "ChangedApiName": "conditionalAccessAllExternalTenants",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **conditionalAccessAllExternalTenants** resource type.",
+			  "Target": "conditionalAccessAllExternalTenants"
+			},
+			{
+			  "Id": "68fb7e36-efee-4e07-a53c-06325d9e1e91",
+			  "ApiChange": "Resource",
+			  "ChangedApiName": "conditionalAccessEnumeratedExternalTenants",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **conditionalAccessEnumeratedExternalTenants** resource type.",
+			  "Target": "conditionalAccessEnumeratedExternalTenants"
+			},
+			{
+			  "Id": "68fb7e36-efee-4e07-a53c-06325d9e1e91",
+			  "ApiChange": "Resource",
+			  "ChangedApiName": "conditionalAccessExternalTenants",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **conditionalAccessExternalTenants** resource type.",
+			  "Target": "conditionalAccessExternalTenants"
+			},
+			{
+			  "Id": "68fb7e36-efee-4e07-a53c-06325d9e1e91",
+			  "ApiChange": "Resource",
+			  "ChangedApiName": "conditionalAccessGuestsOrExternalUsers",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **conditionalAccessGuestsOrExternalUsers** resource type.",
+			  "Target": "conditionalAccessGuestsOrExternalUsers"
+			},
+			{
+			  "Id": "68fb7e36-efee-4e07-a53c-06325d9e1e91",
+			  "ApiChange": "Property",
+			  "ChangedApiName": "excludeGuestsOrExternalUsers",
+			  "ChangeType": "Addition",
+			  "Description": "Added the **excludeGuestsOrExternalUsers** resource type.",
```

Please use the same format for the other property added to the changelog:
```suggestion
			  "Description": "Added the **excludeGuestsOrExternalUsers** property to the [conditionalaccessusers](https://docs.microsoft.com/en-us/graph/api/resources/conditionalaccessusers?view=graph-rest-beta) resource type.",
```